composer.json

```diff
@@ -10,6 +10,9 @@
     "autoload": {
         "psr-4": {
             "Sideways\\Classic\\": "src/"
-        }
+        },
+        "files": [
+            "src/custom_helpers.php"
+        ]
```

Comment:
I think we can create one or more custom "helper" classes with static methods, like

```php
use Sideways\Classic\Helper;

Helper::get_site_schema_markup()
```

This avoids needed to prefix functions and composer's autoload.php will take care of loading the class when needed.

Reply:
I like that, refactorized